src/deferred.js

```diff
@@ -37,7 +37,7 @@ function adoptValue( value, resolve, reject ) {
 	// For Promises/A+, convert exceptions into rejections
 	// Since jQuery.when doesn't unwrap thenables, we can skip the extra checks appearing in
 	// Deferred#then to conditionally suppress rejection.
-	} catch ( /*jshint -W002 */ value ) {
```

Comment:
This is https://jslinterrors.com/value-of-a-may-be-overwritten-in-ie8 rule and we do not support IE8 now, so the rule is redundant at least for that env, but not sure for Android